src/pykeen/datasets/cli.py

```diff
@@ -205,5 +215,77 @@ def verify(dataset: str):
     click.echo(df.to_markdown())
 
 
+@main.command()
+@verbose_option
+@click.option("-d", "--dataset", help="Regex for filtering datasets by name")
+@click.option("-m", "--max-triples", type=int, default=None)
+def expected_metrics(dataset: str, max_triples: Optional[int]):
+    """Compute expected metrics for all datasets (matching the given pattern)."""
+    directory = PYKEEN_DATASETS
+    df_data: List[Tuple[str, str, str, str, float]] = []
+    for _dataset_name, dataset_cls in _iter_datasets(regex_name_filter=dataset, max_triples=max_triples):
+        if dataset_cls is OGBWikiKG:
+            click.echo("Skip OGB WikiKG")
+            continue
+        dataset_instance = get_dataset(dataset=dataset_cls)
+        dataset_name = dataset_instance.__class__.__name__.lower()
```

Comment:
use the dataset resolver for calculating this?

Reply:
81fadc7